Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -228,7 +228,7 @@ class LoginsHelper: TabHelper {
             }
 
             let json = JSON(jsonObj)
-            let src = "window.__firefox__.logins.inject(\(json.rawString()))"
+            let src = "window.__firefox__.logins.inject(\(json.rawString()!))"
```

Comment:
I was also concerned about the crash but I saw the explicit construction just above so didn't worry about it. But I agree it doesn't hurt to be defensive though we may want to log or assert as if it does crash something really bad will probably happen soon (out of memory?).